client/styles/components/_loader.scss

```diff
@@ -0,0 +1,36 @@
+.lds-ring {
```

Comment:
also, the scss should use BEM and should not use pixel measurements (should use rem's).